Locals.lua

```diff
@@ -0,0 +1,23 @@
+local getLocals = "getLocals" -- fix bug
+
+function Locals.getLocals(this)
+    return this
+end
+function Locals.local(this, name)
+    return this[name]
+end
+function Locals.set(this, name, value)
+    this[name] = value
+    return value, name
+end
+function Locals.new()
+    return setmetatable({
+        getLocals = Locals[getLocals];
+        ["local"] = Locals["local"],
+        __index = getLocals
+    }, Locals)
+end
+
+local locals = Locals.new()
+locals.loclas = Locals
```

Comment:
A monad is a monoid in the category of endofunctors. What's the problem?!

Reply:
---